### ISSUE TITLE
strange thing with multiple virtual buttons all using the same senderid whatever I tried

### ISSUE THREAD

**Issue by commentsonly** [mak-gitdev/HA_enoceanmqtt#46]
followed the Wiki.
2 virtual buttons
stick or the base or whatever you'd call is:
Virtual @FFB62B00
![image](https://user-images.githubusercontent.com/35866491/220702698-e1ba68c3-71dd-4532-8b32-348cade7200c.png)

my 2 virtual buttons are defined
[virt_4way_1]
address         = 0xFFFFFFFF
rorg            = 0xF6
func            = 0x02
type            = 0x01
sender          = 0xFFB62B10
virtual         = 1

[virt_4way_2]
address         = 0xFFFFFFFF
rorg            = 0xF6
func            = 0x02
type            = 0x01
sender          = 0xFFB62B11
virtual         = 1
![image](https://user-images.githubusercontent.com/35866491/220702853-3b2c91ca-8126-43dc-bf46-958580cac499.png)

Result: whichever button I do press (AI AO BI BO or released do come from 0xFFB62B11 when looking Eltako PCT14

![image](https://user-images.githubusercontent.com/35866491/220702625-35c9716e-e10a-4483-9bd3-73bf64694b7c.png)

tried with settings>addon>Encean>rebuild .. fixed nothing

And I would have removed the devices if on this delete button per device would be there as it was begore 0.1.24 .. now I can't remove the devices any more.

Perhaps a differnt of 1 isn't enough for the senderid ?

**Comment by mak-gitdev** [comment]
Hi @commentsonly,

> And I would have removed the devices if on this delete button per device would be there as it was begore 0.1.24 .. now I can't remove the devices any more.

Read carefully #17 on 2023/02/17, to see how to delete devices from version 0.1.24.

> Perhaps a differnt of 1 isn't enough for the senderid ?

No, that should be OK.

> Result: whichever button I do press (AI AO BI BO or released do come from 0xFFB62B11 when looking Eltako PCT14

Could you share your enoceanmqtt.log file ?

**Comment by commentsonly** [comment]
found the bug ... dammit sorry for taking up your time. 😄 
the errors sits as always in front of the keyboard.
Perhaps I should quit with names such as virt_4way_1 and virt_4way_2 ... and better use something such as "hoildays" and "mybelovedbutton" so that these differ by more but just a number.

But to let you have something for amusement ..... I got that far by now.
![Animation](https://user-images.githubusercontent.com/35866491/220714393-9d69b2b8-0880-4f3e-9249-90001f6dcf87.gif)
I'd have to work on code-deduplication but I failed till now.
Means 1 shutter-card (slightly modified to get these dammit buttons hidden, who the hell does use 3 buttons nowadays)
And 2 custom buttoncards below for each shutter.
Works ok .. well work as the wallbuttons so. 1 click and the shutter starts rolling another click and it does stop. therefore 2 buttons one for stop/open the other for stop/close.
The real nice one with the enocean addon is I mustn't care about switches to be driven since sending telegrams does work. And since so the shutter card also move when pessing when physical button without the need to use that cover-time-based-sync integration.

Where's the voting page for the best HA addon? 😄
